src/java/fr/paris/lutece/portal/service/security/LuteceUser.java

```diff
@@ -145,7 +146,7 @@
     private String _strAuthenticationService;
 
     /** Authentication Service impl */
-    private LuteceAuthentication _luteceAuthenticationService;
+    private transient LuteceAuthentication _luteceAuthenticationService;

```

Comment:
This does not make the class properly serializable. On reactivation, this field will be `null`

Reply:
will be rollbacked